docs/source/model_design/vivarium_model_components/causes/index.rst

```diff
@@ -865,5 +866,11 @@ Incidence- vs. Prevalence-Based YLDs
      - YLD and YLL measurements are measured using different methods; For diseases with a short duration, YLDs may be underestimated.
 
 
+YLDs in Vivarium 
+^^^^^^^^^^^^^^^^
+DWs and YLDs in Vivarium apply to the duration of the condition. Remember, YLDs are a product of the DW associated with the condition and the time between onset and survival. To illustrate an example, let's say an individual contracted the flu with a disability weight of 0.05 and recovered after 1 week. To calculate the YLDs here, we would multiply 0.05 by 1/52 (0.019).
```

Comment:
It also might be nice to incorporate some information about vivarium timesteps here. How it would work in vivarium is that if we had timesteps of one week, then the simulant would accumulate YLDs equal to DW * time_step for each timestep that they are infected, where timestep is defined in a fraction of one year. Therefore, choosing an appropriate timestep duration is important for getting YLDs correct! If we had month long timesteps, then a case of the flu (which should really only be 1 week) would accrue YLDs for the flu over an entire month.

Reply:
^ This implies that (YLDs_cause_1_each_timestep + YLDs_cause_2_each_timestep) > YLDs_overall_each_timestep

Reply:
@alibow I wasn't able to totally get to the bottom of it, but I am reasonably confident from [the Vivarium code](https://github.com/ihmeuw/vivarium_public_health/blob/d8db587f56e5ffcba97fd6bb77fd2d2ff585218d/src/vivarium_public_health/metrics/disability.py#L175-L182) that you're correct about how Vivarium calculates cause-specific and overall YLDs.